src/sentry/issues/ingest.py

```
@@ -70,6 +72,15 @@ def save_issue_occurrence(
             group_info.group.project, environment, release, [group_info]
         )
         _get_or_create_group_release(environment, release, event, [group_info])
+
+        # Create IncidentGroupOpenPeriod relationship for metric issues
+        if occurrence.type == MetricIssue:
+            open_period = get_latest_open_period(group_info.group)
+            if open_period:
```

Should this fail loudly?